addons/FlashCards/src/presenter.js

```diff
@@ -386,6 +386,7 @@ function AddonFlashCards_create(){
         return !$(presenter.$view.find(".flashcards-card").get(0)).hasClass("flashcards-card-reversed");
     }
 
+    presenter.originalCard = -1;
     presenter.prevCard = function () {
         if ((presenter.isFrontPlaying || presenter.isBackPlaying) && presenter.state.currentCard > 1) {
             presenter.sendEndedEvent();

```

Comment:
biorąc pod uwagę proponowaną zmianę w presenter.nextCard to tutaj też można przekazywać presenter.originalCard